hack/gen-content.sh

```diff
@@ -82,8 +82,8 @@ find_md_files() {
 #   Links:
 #   ./bug-bounty.md -> /guide/bug-bounty
 #   contributor-cheatsheet/README.md -> /guide/contributor-cheatsheet
-#   ../../sig-list.md -> https://github.com/kubernetes/community/blob/master/sig-list.md
-#   /contributors/devel/README.md -> https://github.com/kubernetes/community/blob/master/contributors/devel/README.md
+#   ../../sig-list.md -> https://git.k8s.io/community/sig-list.md
+#   /contributors/devel/README.md -> https://git.k8s.io/community/contributors/devel/README.md
```

Comment:
One small thing, this is an example the link expander. This should probably go back or what it was previously as it expands to the GitHub link independent of the org and git.k8s.io afaik is only for the kubernetes org

Otherwise lgtm 👍 


(could also be done in a follow up PR)

Reply:
OK I reverted this change in a separate commit explaining why